cpptod.dd

```diff
@@ -829,6 +830,84 @@ void test()
 }
 ------
 
+<hr>$(COMMENT  -------------------------------------------- )
+
+$(H3 <a name="interfaces">Interfaces</a>)
+
+Interfaces are used to describe the behavior or capabilities of a class without
+commiting to a particular implementation of the class.
+
+$(H4 The C++ Way)
+
+  In C++, an interface is implemented with an `Abstract Class`. The abstract class
```

Comment:
Only things that might appear in code should come in code font. "Abstract Class" is not code. It may be emphasized by means of $(I ...)